README.md

```diff
@@ -204,6 +204,25 @@ If the implicit vote is not overwritten with an explicit vote, the validity of t
 requirements as any other vote. For the purpose of validity checking, it can be considered an unwritten comment placed
 after all comments on the PR before the merging comment of the PR.
 
+**330** *Conditional Voting*
+
+A player may cast a conditional vote by posting a comment in the following format:
+
+```
+IF [condition] THEN [voting post] (ELSE [voting post])
+```
+
+In this format, items in between parentheses are optional. `[condition]` should
+be replaced with a human readable condition. If this condition cannot be resolved
+at a time when a player attempts to do so, for example because of improper formulation or
+a logical paradox, it will be resolved as being false.
+
+`[voting post]` should be replaced by the contents of a comment representing the vote,
+or an empty string. Conditional votes are a comment representing a vote.
+
+In order to perform an action based on resolution of one of more conditional votes,
+a player must post a comment describing the resolution before performing said action.
```

Comment:
If two players consider it to be different results, then the condition cannot be resolved, and it should be false. So if A were to merge a PR, for which the truthiness is required, then, yes, they might be put on trial.